CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## v0.3.1 - 2022-08-24
+
+- Fix error when encoding integer key.
```

Comment:
This was not an error but instead it was something that was not supported. Could you change this to describe what the new feature is please? Thank you

Reply:
Can it be "Adds support for integer keys."?